backend/alertmanager/alertmanager.yml

```
@@ -37,51 +37,18 @@ receivers:
       - url: 'http://backend:443/api/v1/health'
-        send_resolved: true
-        title: '⚠️ WARNING: {{ .GroupLabels.alertname }}'
-        text: |
-          Warning condition detected - monitoring recommended.
-          
-          {{ range .Alerts }}
-          Alert: {{ .Annotations.summary }}
-          Description: {{ .Annotations.description }}
-          {{ end }}
\ No newline at end of file
+        send_resolved: true
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)